### PR TITLE
Prepare for 24.10 Release

### PR DESCRIPTION
**Pull request by jdye64**
## Description
Preparation for 24.10 release

## Checklist
- [ ] I am familiar with the [Contributing Guidelines](https://github.com/NVIDIA/nv-ingest/blob/main/CONTRIBUTING.md).
- [ ] New or existing tests cover these changes.
- [ ] The documentation is up to date with these changes.

### CODE REVIEW

**Review by drobison00 — APPROVED** [review]

**Comment by sosahi** [comment]
before merging please:

1. update the embedding NIM version
2. pull in the latest changes/fixes

File: docker-compose.yaml [hunk 1]
```diff
@@ -95,7 +95,6 @@ services:
     runtime: nvidia
 
   embedding:
-    # NIM ON
     image: nvcr.io/nim/nvidia/nv-embedqa-e5-v5:1.0.1
```
Comment (randerzander) [inline docker-compose.yaml]:
```suggestion
    image: nvcr.io/nim/nvidia/nv-embedqa-e5-v5:1.1.0
```

File: README.md [hunk 2]
```diff
@@ -134,13 +134,13 @@ NV_INGEST_ROOT=<PATH_TO_THIS_REPO>
...
+02c9ab8c6901   nvcr.io/ohlfw0olaadg/ea-participants/cached:0.2.0                          "/opt/nvidia/nvidia_…"   14 hours ago     Up 24 seconds             0.0.0.0:8006->8000/tcp, :::8006->8000/tcp, 0.0.0.0:8007->8001/tcp, :::8007->8001/tcp, 0.0.0.0:8008->8002/tcp, :::8008->8002/tcp                                                                                                                                                      nv-ingest-cached-1
 d49369334398   nvcr.io/nim/nvidia/nv-embedqa-e5-v5:1.0.1                                  "/opt/nvidia/nvidia_…"   14 hours ago     Up 33 seconds             0.0.0.0:8012->8000/tcp, :::8012->8000/tcp, 0.0.0.0:8013->8001/tcp, :::8013->8001/tcp, 0.0.0.0:8014->8002/tcp, :::8014->8002/tcp                                                                                                                                                      nv-ingest-embedding-1
```
Comment (randerzander) [inline README.md]:
```suggestion
d49369334398   nvcr.io/nim/nvidia/nv-embedqa-e5-v5:1.1.0                                  "/opt/nvidia/nvidia_…"   14 hours ago     Up 33 seconds             0.0.0.0:8012->8000/tcp, :::8012->8000/tcp, 0.0.0.0:8013->8001/tcp, :::8013->8001/tcp, 0.0.0.0:8014->8002/tcp, :::8014->8002/tcp                                                                                                                                                      nv-ingest-embedding-1
```

**Comment by randerzander** [comment]
I'm guessing the Helm chart needs an update to the latest embedding 1.1.0 version as well

File: docker-compose.yaml (hunk 1 above)
Comment (jdye64) [reply docker-compose.yaml]: done

File: README.md (hunk 2 above)
Comment (jdye64) [reply README.md]: done

**Comment by jdye64** [comment]
Actually the Helm charts do not currently deploy the embed model. We should slate that for the next release however.

**Pull request merged**